src/app/modules/timesheet/components/timesheet-edit/timesheet-edit.component.html

```diff
@@ -1,160 +1,152 @@
-      </div>
-
-      <div class="edit-cra__wrapper__form__component">
-        <label class="edit-cra__wrapper__form__component__toggle">
-          <input name="invoiceToggle" type="checkbox" [(ngModel)]="generateInvoice" #invoiceToggle>
-          Générer une facture
-        </label>
-        <app-invoice-form *ngIf="generateInvoice" [invoice]="timesheetService.timesheet.invoice"
-          (changed)="onUserInput()"></app-invoice-form>
-      </div>
-
-      <div class="edit-cra__wrapper__form__message" *ngIf="submitMessage"
-        [ngClass]="'edit-cra__wrapper__form__message--' + submitMessage.type">
-        <u>{{ submitMessage.text }}</u>
-      </div>
-
-      <div class="edit-cra__wrapper__form__submit">
-        <button data-text="Valider mon CRA" class="edit-cra__wrapper__form__submit__button" (click)="onSubmit()">
-          <span>Valider mon CRA</span>
-        </button>
-      </div>
-
-      <div *ngIf="showLinks" class="edit-cra__wrapper__form__icons">
-        <div *ngIf="generateInvoice" class="edit-cra__wrapper__form__icons__item">
-          <a target="_self" [href]="timesheetService.getInvoiceLink()">
-            <i class="material-icons">picture_as_pdf</i>
-            <p>Télécharger la facture</p>
-          </a>
-        </div>
-        <div class="edit-cra__wrapper__form__icons__item">
-          <a href="#" [mailto] [mailContent]="reviewMail">
-            <i class="material-icons">email</i>
-            <p>Envoyer le CRA par mail</p>
-          </a>
-        </div>
-        <div class="edit-cra__wrapper__form__icons__item">
-          <a href="#" [copyToClipboard]="originUrl + '/timesheet/edit/' + timesheetService.getEditToken()"
-            [copyToClipboardMessage]="'Lien copié'" [copyToClipboardMessageParent]="editLink.parentNode" #editLink>
-            <i class="material-icons">edit</i>
-            <p>Copier le lien de modification</p>
-          </a>
-        </div>
-        <div class="edit-cra__wrapper__form__icons__item">
-          <a href="#" [copyToClipboard]="originUrl + '/timesheet/review/' + timesheetService.getReviewToken()"
-            [copyToClipboardMessage]="'Lien copié'" [copyToClipboardMessageParent]="shareLink.parentNode" #shareLink>
-            <i class="material-icons">remove_red_eye</i>
-            <p>Copier le lien de partage</p>
-          </a>
-        </div>
-      </div>
-    </form>
-  </div>
-</div>
+<div class="edit-cra">
+  <div class="edit-cra__wrapper">
+    <div class="edit-cra__wrapper__title">
+      <h2>{{ getModeTitle() }} un compte rendu d'activité</h2>
+    </div>
+
+    <form class="edit-cra__wrapper__form" #form="ngForm">
+      <div id="prestataire" class="edit-cra__wrapper__form__block">
+        <h4 class="edit-cra__wrapper__form__block__title">Consultant</h4>
+        <div class="edit-cra__wrapper__form__block__input-wrapper">
+          <div class="edit-cra__wrapper__form__block__input-wrapper__item form-group">
+            <label for="consultantNameInput"
```

Comment:
Adapt all class, label... according to your previous naming choice :)

Reply:
Cannot respect Kebab case because: "-" is not allowed in reference names